odoo/addons/test_lint/tests/_odoo_checker_sql_injection.py

```diff
@@ -105,15 +111,16 @@ def _check_sql_injection_risky(self, node):
             current = node
             while (current and not isinstance(current.parent, astroid.FunctionDef)):
                 current = current.parent
-            parent = current.parent
-
-            # 2) check how was the variable built
-            for node_ofc in parent.nodes_of_class(astroid.Assign):
-                if node_ofc.targets[0].as_string() != first_arg.as_string():
-                    continue
-                is_concatenation = self._check_concatenation(node_ofc.value)
-                if is_concatenation:
-                    break
+            if current:
+                parent = current.parent
+
+                # 2) check how was the variable built
+                for node_ofc in parent.nodes_of_class(astroid.Assign):
```

Comment:
Since you're rewriting this bit (well reindenting but shows up the same on default diff), `node_ofc` should be renamed to `assignment`, as the reader would be less interested by knowing it's a node of a class, and more by what kind of node it is.